pkg/asset/installconfig/vsphere/vsphere.go

```diff
@@ -104,7 +104,9 @@ func getClients() (*vCenterClient, error) {
 				Message: "vCenter",
 				Help:    "The hostname of the vCenter to be used for installation.",
 			},
-			Validate: survey.Required,
+			Validate: survey.ComposeValidators(survey.Required, func(ans interface{}) error {
+				return validate.ClusterName1035(ans.(string))
```

Comment:
This check is not quite what we need.
1) It is valid for the vCenter hostname to be an IP address.
1) We should not validate the length of the hostname.